packages/audioplayers/tizen/src/audio_player.cc

```diff
@@ -0,0 +1,484 @@
+#include "audio_player.h"
+
+#include "audio_player_error.h"
+#include "log.h"
+
+static std::string StateToString(player_state_e state) {
+  std::string result;
+  switch (state) {
+    case PLAYER_STATE_NONE:
+      result = "PLAYER_STATE_NONE";
+      break;
+    case PLAYER_STATE_IDLE:
+      result = "PLAYER_STATE_IDLE";
+      break;
+    case PLAYER_STATE_READY:
+      result = "PLAYER_STATE_READY";
+      break;
+    case PLAYER_STATE_PLAYING:
+      result = "PLAYER_STATE_PLAYING";
+      break;
+    case PLAYER_STATE_PAUSED:
+      result = "PLAYER_STATE_PAUSED";
+      break;
+  }
+  return result;
+}
+
+static std::string PlayerErrorToString(int code) {
```

Comment:
What a long list of errors. You may apply @bbrto21's solution here. https://github.com/flutter-tizen/plugins/pull/60#discussion_r608352262

(and hopefully to other plugins as well if you have time)

Reply:
I've updated the code with the approach mentioned in #60. Indeed the code is a lot shorter now. 
I will apply the same solution to other plugins after this pr gets merged.